test/testcases/wrapping/wrapping_metadata.hxtest

```diff
@@ -0,0 +1,60 @@
+abstract OpenGLES2RenderContext(OpenGLES3RenderContext) from OpenGLES3RenderContext
+{}
+
+---
+
+@:forward(ACTIVE_ATTRIBUTES, ACTIVE_TEXTURE, ACTIVE_UNIFORMS, ALIASED_LINE_WIDTH_RANGE, ALIASED_POINT_SIZE_RANGE, ALPHA, ALPHA_BITS, ALWAYS, ARRAY_BUFFER,
```

Comment:
Please tell me this isn't actual code you found somewhere... :D

Reply:
found it in lime... :)